packages/hoverable/src/with-hover.hoc.tsx

```diff
@@ -5,7 +5,12 @@ import { HoverProps } from './hover.props';
 
 const isHoverableReactElement = (el: Maybe<ReactElement>) =>
     el !== null &&
-    (el.props?.hoverStyle !== undefined || el.props?.activeStyle !== undefined || el.props?.hoverColor !== undefined);
+    (el.props?.hoverStyle !== undefined ||
```

Comment:
btw, why is there `||`, I think, if nested component is hoverable, then it should have all of such props and there should be `&&`

@vitalyiegorov can you put some light here please?
Because I think @AndrewJimson did this according to your previous code